docs/topics/auth/customizing.txt

```diff
@@ -299,6 +299,32 @@ example, the following checks if a user may view tasks::
 
     user.has_perm('app.view_task')
 
+.. versionadded:: 1.11
+
+Custom permissions may contain the ``%(class)s`` string. Child models will then
+inherit the specific permission::
+
+    class BaseTask(models.Model):
+        ...
+        class Meta:
+            abstract = True
+            permissions = (
+                ("view_%(class)s", "Can view %(class)s"),
+            )
+
+
+    class Task(BaseTask):
+        ...
+        class Meta:
+            permissions = (
```

Comment:
Is this example correct? I thought you had to use `BaseTask.Meta.permissions` as in the test?